### ISSUE TITLE
[FEATURE] Chineses translations

### ISSUE THREAD

**Issue by Dustyposa** [tiangolo/fastapi#1228]
### Is your feature request related to a problem

Is your feature request related to a problem?

I want to be able to track chapters that are in progress or already translated.

### The solution you would like

Each person who is working in a chapter should notify in this issue. Comment with the name of the file. If more than two weeks are passed, other people can take this file.

### Describe alternatives you've considered

We use notion to distribution chapter and tracking translating  sate.
If you want join our, you can contact with us.  @waynerv @RunningIkkyu 

### Additional context
It ’s okay to talk about this in this issue.

**Comment by alicrazy1947** [comment]
https://github.com/tiangolo/fastapi/pull/2261

**Comment by lpdswing** [comment]
Add chinese translate for tutorial-middleware. #2334

**Comment by maoyibo** [comment]
chinese translate python-types.md Add `Generic types` Fix `Optional` #2360

**Comment by maoyibo** [comment]
chinese translation add tutorial/sql-databases.md #2379

**Comment by maoyibo** [comment]
docs/zh query_params.md rm superfluous content #2384
The last chapter "Optional 类型声明" not exists any more in English tutorial, and tutorial007.py not exists too. so this chapter should be removed.

**Comment by maoyibo** [comment]
docs/zh body.md fix code highlight #2385
Fix code highlight lines and add two tips.

**Comment by maoyibo** [comment]
chinese translation add tutorial/sql-databases.md #2379 Add body-nested-models.md

**Comment by lpdswing** [comment]
可以互相review发自我的华为手机-------- 原始邮件 --------发件人： maoyibo <notifications@github.com>日期： 2020年11月24日周二 下午3:40收件人： tiangolo/fastapi <fastapi@noreply.github.com>抄送： lpdswing <lpdswing@qq.com>, Comment <comment@noreply.github.com>主    题： Re: [tiangolo/fastapi] [FEATURE] Chineses translations (#1228)
chinese translation add tutorial/sql-databases.md #2379 Add body-nested-models.md

**Comment by maoyibo** [comment]
Chinese translation add tutorial/sql-databases #2408
Chinese translation add tutorial/body-nested-models #2409
Chinese translation add tutorial/extra-data-types #2410
Chinese translation add tutorial/schema-extra-example #2411
Chinese translation add tutorial/header-params #2412

**Comment by maoyibo** [comment]
@waynerv  Add Chinese translation for body-nested-models.md #1609

**Comment by waynerv** [comment]
@maoyibo Thanks.There are still many translated chapters not mentioned in this issue.

Check the currently existing pull requests before you starting a new chapter.

**Comment by waynerv** [comment]
I'll complete next 3 chapers in this week:
- tutorial/reponse-model.md
- tutorial/extra-models.md
- tutorial/reponse-status-code.md

@maoyibo @lpdswing Is there anyone already working on this?

**Comment by maoyibo** [comment]
@waynerv I have already translated half of tutorial/reponse-model.md. How is your progress?

**Comment by waynerv** [comment]
@maoyibo Already done.....Can you email your wechat ID to me?

**Comment by maoyibo** [comment]
> @maoyibo Already done.....Can you email your wechat ID to me?

Ok I will review you PR.

**Comment by maoyibo** [comment]
All Chinese translation PR not in this list:
Add Chinese translations for advance/index.md #1445
Add Chinese docs for advanced/path-operation-advanced-configuration.md #1447
Add cn translation for serveral sections of advanced user guide. #1804 
Add Chinese translation for benchmarks.md #2119

**Comment by maoyibo** [comment]
🌐 Add Chinese translation for Tutorial - Form Data #2420
@waynerv and other friends please take a look.

**Comment by maoyibo** [comment]
I recently translated the chapter of Dependencies. There are some glossary that I'm not sure of translating accurately. Please review it. @waynerv and other friends.
Dependencies - First Steps #2436
Classes as Dependencies #2437
Sub-dependencies #2438
Dependencies in path operation decorators #2439
Global Dependencies #2440
Dependencies with yield #2441

**Comment by waynerv** [comment]
I'll complete these 5 chapters in next 2 weeks:
-  tutorial/security/first-steps.md
-  tutorial/security/get-current-user.md
-  tutorial/security/index.md
-  tutorial/security/oauth2-jwt.md
-  tutorial/security/simple-oauth2.md

**Comment by blt232018** [comment]
我现在正在学习 FastAPI，也想为翻译中文文档提供一些帮助。我可以加入你们吗？
 
![image](https://user-images.githubusercontent.com/43393471/101122828-5822c000-362e-11eb-9461-09e0f007dbb8.png)

**Comment by waynerv** [comment]
@blt232018 Hi，欢迎加入 FastAPI 文档的中文翻译，你只需要按照文档的指引提交 PR 就可以了，另外在开始前将你想翻译的章节发到这个 issue 里面，以防其他人重复翻译。也欢迎你 review 已经提交的中文翻译 PR。

**Comment by blt232018** [comment]
> @blt232018 Hi，欢迎加入 FastAPI 文档的中文翻译，你只需要按照文档的指引提交 PR 就可以了，另外在开始前将你想翻译的章节发到这个 issue 里面，以防其他人重复翻译。也欢迎你 review 已经提交的中文翻译 PR。

@waynerv 谢谢！现在有没有 FastAPI 文档中文翻译专门的微信群或者其它方便即时交流的途径？如果有的话请告诉我。
我当前在本地翻译了 Security 之前的全部章节（notebook格式，还没有转换成markdown），但是看到这里好像已经完成了。

有一个问题是，有些已经完成翻译的的章节并不是按照文档顺序。所以有必要有一个即时交流的群，定期汇总更新未翻译的章节以及研究讨论翻译中遇到的问题，这样不用每次花费时间翻看所有的评论来确定自己打算翻译的章节是不是已经完成了。

**Comment by waynerv** [comment]
@blt232018 你把你的微信号发我邮箱吧，我加你😄
ampedee@gmail.com

**Comment by blt232018** [comment]
> @blt232018 你把你的微信号发我邮箱吧，我加你😄
> ampedee@gmail.com

@waynerv 谢谢，我已经发邮件给你了。😁😁😁

**Comment by blt232018** [comment]
- [ x] tutorial/cors.md #2540

**Comment by TimorChow** [comment]
Hi, I am interested in FastAPI, 希望加入中文翻译组, 帮主翻译和review, 请问现在参与可以吗?
@waynerv @RunningIkkyu @maoyibo

**Comment by ghandic** [comment]
What are all these notifications for?

**Comment by pylover7** [comment]
给小伙伴们做了个镜像文档 https://fastapi.pylover.net
为了防止有时候官方文档访问不了的情况~
（反正我有时候会访问出问题，所以自己做了一个分享给大家，和官方的一摸一样的，有需要的自取~）

**Comment by Jedore** [comment]
I want to join and and have sent you a email. @waynerv

**Comment by Jedore** [comment]
看了几个相对较新的中文翻译PR，还存在着一些语法、排版、甚至语句不是很通顺、更不用说有一些概念翻译的不对的了，但是居然都 approve 了。 本来也是想着参与翻译，攒个贡献，但也得认真仔细，不然真的是会误导许多人。没想到居然有翻译、审阅都不认真的，瞬间打消了不少积极性！！！

**Comment by mkdir700** [comment]
I will translate the following chapters.
- [ ] async.md

**Comment by Lenclove** [comment]
Hi, everyone! I would like to help translations with FastAPI. So is there any translations or PR that I can help?

**Comment by tiangolo** [comment]
Yep! You could help review these PRs: https://github.com/tiangolo/fastapi/pulls?q=is%3Aopen+is%3Apr+label%3Alang-zh+label%3A%22awaiting+review%22

**Comment by Jedore** [comment]
> Hi, everyone! I would like to help Chinese translations with FastAPI. So is there any translations or PR that I can help?

this one https://github.com/tiangolo/fastapi/pull/5591

**Comment by Jedore** [comment]
> Yep! You could help review these PRs: https://github.com/tiangolo/fastapi/pulls?q=is%3Aopen+is%3Apr+label%3Alang-zh+label%3A%22awaiting+review%22

How to deal with PRs like this https://github.com/tiangolo/fastapi/pull/3815#issuecomment-1288184987, the author may be offline.
